homeassistant/components/zwave/__init__.py

```diff
@@ -627,6 +628,39 @@ def value_changed(self, value):
         self.update_properties()
         self.schedule_update_ha_state()
 
+    def value_handler(self, method='get', class_id=None, index=None,
```

Comment:
My preference would be to make this an internal method, and add wrapper functions `get_value` and `set_value`. I think it's quicker to read and understand `self.get_value`/`self.set_value` in the code instead of `self.value_handler`.